app/helpers/application_helper.rb

```diff
@@ -597,4 +597,11 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def speaker_selector_input(form)
+    users = User.select(:id, :name, :email).where(is_disabled: false).order(:name)
```

Comment:
Using `.select` will instantiate `User` objects. When we have a lot of users in the DB, that will eat up a lot of memory. Use `.pluck` instead. That will give you an array of arrays, which is lighter.